SRS.md

```diff
@@ -142,43 +138,23 @@ This use case covers the default view and contains the existing exams.
 
 [Show Dashboard](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_ShowDashboard.md)
 
-#### 3.1.5 Update Progress Bar
-
-There are three states for the topics. Pending, started and finished. These are calculated based on the saved progress and the time spent. For the user, the current state is indicated by a corresponding background color.
-
-[Update Progress Bar](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_UpdatedProgressBar.md)
-
-#### 3.1.6 Send Notifications
+#### 3.1.5 Start/Stop Timer
 
-To inform and acitivate the user to learn, the systems sends push up messages to the users smartphone.
+The estimated time can be count down by a start Timer button. As well the Timer can be stopped by another button.
 
```

Comment:
The estimated time can be counted down by a start Timer button. As well, the Timer can be stopped by another button.